packages/web3-core/src/web3_context.ts

```diff
@@ -365,6 +366,33 @@ export class Web3Context<
 			this._requestManager as unknown as Web3RequestManager,
 		);
 	}
+
+	/**
+	 * This method allows extending the web3 modules.
+	 * Note: This method is only for backward compatibility, and It is recommended to use Web3 v4 Plugin feature for extending web3.js functionality if you are developing some thing new.
```

Comment:
```suggestion
	 * Note: This method is only for backward compatibility. And it is recommended to utilize the Web3 v4 Plugin feature for extending web3.js functionality, especially if you are one who provides the extended RPC methods.
```
I think it would be good like this if you liked it.